docs/usage/shoot_high_availability_best_practices.md

```diff
@@ -0,0 +1,431 @@
+# Implementing High Availability and Tolerating Zone Outages
+
+Implementing highly available software that can tolerate even a zone outage unscathed is no trivial task. You will find here various recommendations to get closer to that goal. While many recommendations are general enough, the examples are specific in how to achieve this in a Gardener-managed cluster and where/how to tweak the different control plane components. If you do not use Gardener, it may be still a worthwhile read.
+
+First however, what is a zone outage? It sounds like a clear-cut "thing", but it isn't. There are many things can go haywire. Here some examples:
+
+- Elevated cloud provider API error rates for individual or multiple services
+- Network bandwidth reduced or latency increased, usually also effecting storage sub systems as they are network attached
+- No networking at all, no DNS, machines shutting down or restarting, ...
+- Functional issues, of either the entire service (e.g. all block device operations) or only parts of it (e.g. LB listener registration)
+- All services down, temporarily or permanently (the proverbial burning down data center :fire:)
+
+This and everything in between make it hard to prepare for such events, but you can still do a lot. The most important recommendation is to not target specific issues too much - tomorrow another service will fail in an unanticipated way. Rather focus on [meaningful availability](https://research.google/pubs/pub50828) than on internal signals (useful, but not as relevant as the former). Always prefer automation over manual intervention (e.g. leader election is a pretty robust mechanism, auto-scaling may be required as well, etc.).
+
+Also remember that HA is costly - you need to balance it against the cost of an outage as silly as this may sound, e.g. running all this excess capacity "just in case" vs. "going down" vs. a risk-based approach in between where you have means that will kick in, but they are not guaranteed to work (e.g. if the cloud provider is out of resource capacity). Maybe some of your components must run at the highest possible availability level, but others not - that's a decision only you can make.
+
+## Control Plane
+
+The Kubernetes cluster control plane is managed by Gardener (as pods in separate infrastructure clusters to which you have no direct access) and can be set up with no, node, or zone HA.
```

Comment:
I'll try to rewrite that then/follow your advice. I decided at first against it, because it seemed to chatty to me (and didn't include the `none` type), but thinking about comments, I realised that I colored the document with my own mental model and I should have used the API terminology instead.